lib/Mojolicious/Command/Author/generate/callbackery_app/lib/App/GuiPlugin/SongForm.pm

```diff
@@ -107,23 +107,17 @@ has formCfg => sub {
                 structure => $decades
             }
         },
-        (exists $self->{args}{currentFormData}{song_decade} && $self->{args}{currentFormData}{song_decade}) ? {
+       {
             key               => 'song_decade_addional',
             label             => trm('Addional Decade info'),
             widget            => 'text',
             reloadOnFormReset => true,
-            set               => {
-                placeholder => '',
-                readOnly    => false,
-            },
-        } : {
-            key               => 'song_decade_addional',
-            label             => trm('Addional Decade info'),
-            widget            => 'text',
-            reloadOnFormReset => true,
-            set               => {
+            set               => $self->{args}{currentFormData}{song_decade} ? {
```

Comment:
Aren't the if cases the wrong way round now?

Reply:
so true